content/en/news/releases/1.10.x/announcing-1.10.5/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Announcing Istio 1.10.5
+linktitle: 1.10.5
+subtitle: Patch Release
+description: Istio 1.10.5 patch release.
+publishdate: 2021-09-27
```

Comment:
Assume the date will be a few days from now?

Reply:
Updated.